### PR TITLE
[#212] Locale-unspecified message properties to default to 'en' messages.

### PR DESCRIPTION
**Pull request by mogoodrich**
…ffix as Englis

### CODE REVIEW

**Comment by mogoodrich** [comment]
Closing this, this needs to be tweaked/fixed

**Review by mseaton — APPROVED** [review]

**Comment by mogoodrich** [comment]
My original PR preferred English over the default locale set via OpenMRS global property (and caused a test failure).  New PR with what is hopefully the proper fix and a new test.

File: api/src/test/java/org/openmrs/module/initializer/InitializerMessageSourceIntegrationTest.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
  */
 package org.openmrs.module.initializer;
 
+import net.sf.cglib.core.Local;
```
Comment (mseaton) [inline api/src/test/java/org/openmrs/module/initializer/InitializerMessageSourceIntegrationTest.java:12]: Bad import - remove this

**Review by mseaton — CHANGES_REQUESTED** [review]
One minor comment to remove a bad import.  Also should add a bullet point to the release notes describing the change.

**Comment by mogoodrich** [comment]
FYI, looks like the build error is unrelated to this commit? "Error:  Failed to execute goal on project initializer-api-bahmni: Could not resolve dependencies for project org.openmrs.module:initializer-api-bahmni:jar:2.4.1-SNAPSHOT: The following artifacts could not be resolved: org.bahmni.module:bahmnicore-api:jar:0.93-1.1.0, org.openmrs.module:episodes-api:jar:1.0-1.0.0: Could not find artifact org.bahmni.module:bahmnicore-api:jar:0.93-1.1.0 in openmrs-repo (https://mavenrepo.openmrs.org/public) -> [Help 1]"

**Comment by mogoodrich** [comment]
This should be ready to go now @mks-d @mseaton @ibacher

**Review by mseaton — APPROVED** [review]
Looks good to me.  @ibacher / @mks-d - happy to provide some context on this.  Would be good to get this bug fix merged in sooner rather than later.

**Comment by mogoodrich** [comment]
Thanks @mseaton ... yes @ibacher @mks-d  it's a rather serious bug... ironically, it means that if your OS locale is something other than English, when people set their OpenMRS locale to that language, the site generally shows up in English instead.

**Review by mks-d — APPROVED** [review]

**Pull request merged**